cosmo_tester/test_suites/test_snapshots/snapshots_migration_from_321_and_331_to_34_test.py

```diff
@@ -0,0 +1,276 @@
+import json
+import os
+import subprocess
+import tempfile
+
+import pkg_resources
+
+import cosmo_tester
+from cosmo_tester.framework.git_helper import clone
+from cosmo_tester.framework.testenv import (
+    TestCase, initialize_without_bootstrap, clear_environment)
+from cosmo_tester.framework.util import YamlPatcher
+
+MANAGER_KEY_FILE_NAME = 'manager_kp.pem'
+AGENTS_KEY_FILE_NAME = 'agents_kp.pem'
+BLUEPRINTS_REPO = 'https://github.com/cloudify-cosmo/' \
+                  'cloudify-manager-blueprints.git'
+CLI_REPO = 'https://github.com/cloudify-cosmo/cloudify-cli.git'
+HELLO_WORLD_REPO = 'https://github.com/cloudify-cosmo/' \
+                   'cloudify-hello-world-example'
+SNAPSHOT_TOOL_REPO = 'https://github.com/cloudify-cosmo/' \
+                     'cloudify-3.2.1-snapshots-tool'
+
+BRANCH_NAMES = {
+    '3.3.1': '3.3.1-build',
```

Comment:
should probably be 3.3.1 and 3.2.1 rather than the build branches. 
also, lets write 3.4 instead of 3.4rc1